fs_mark/fs_mark.py

```diff
@@ -0,0 +1,159 @@
+import trunner
+import time
+import re
+import pexpect
+import timing_data as t_data
+import psh.tools.psh as psh
+
+from trunner.ctx import TestContext
+from trunner.dut import Dut
+from trunner.types import TestResult, Status
+
+
+def clean(dut, test_name, ctx: TestContext) -> int:
+    # Dummyfs doesn't need clean since target is rebooted after every test
+    if not ctx.target.rootfs:
+        return 0
+
+    if test_name and ('-k' in test_name or '-L' in test_name or '-F' in test_name):
+        args = test_name.split()
+        args = args[1:]
+
+        def get_dir(x):
+            if args[x[0] - 1] == '-d':
+                return x[1]
+
+        dirs = list(map(lambda x: get_dir(x), enumerate(args)))
+        dirs = [x for x in dirs if x is not None]
+
+        dut.expect(re.escape(ctx.target.shell_prompt), timeout=120)
+        psh._send(dut, f'/bin/fs_mark_clean {" ".join(dirs)}')
```

Comment:
So, if we're using dut everywhere, let's use it here too instead of importing psh